File: lib/models/user_model.dart
class UserModel {
  String userid;
  String username;
  String password;
  String email;
  String avatarUrl;
  String phoneNumber;
  DateTime birthday;
  String type;
  String token;
  String renewalToken;
  UserModel(
      {this.userid,
      this.username,
      this.password,
      this.email,
      this.avatarUrl,
      this.phoneNumber,
      this.birthday,
      this.type,
      this.token,
      this.renewalToken});
  factory UserModel.fromJson(Map<String, dynamic> responseData) {
    return UserModel(
        userid: responseData['id'],
        username: responseData['name'],
        email: responseData['email'],
        phoneNumber: responseData['phone'],
        type: responseData['type'],
        token: responseData['access_token'],
        renewalToken: responseData['renewal_token']);
  }
  UserModel.userLogin({String userName, String passWord}) {
    this.username = userName;
    this.password = passWord;
  }
  UserModel.defaultUser(Map userData) {
    username = userData['username'];
    password = userData['password'];
  }
  UserModel.userRegistor(String userName, String passWord, String email) {
    this.username = userName;
    this.password = passWord;
    this.email = email;
  }
}

File: Api/QR_Code_Api/QR_Code_Api/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace QR_Code_Api.Models
{
    public class UserModel
    {
        public string UserId { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string AvatarUrl { get; set; }
        public string PhoneNumber { get; set; }
        public DateTime Birthday { get; set; }
        public string Type { get; set; }
        public string Token { get; set; }
        public string RenewalToken { get; set; }
    }
}
